lighthouse-core/report/html/templates.html

```diff
@@ -362,6 +367,7 @@
     */
     .pyro {
       display: none;
+      z-index: 1;
```

Comment:
we need `pointer-events: none` now. it's hard but possible to click on the fireworks div blocking the click on the gauge